src/friends.rs

```diff
@@ -316,6 +316,32 @@ unsafe impl Callback for GameLobbyJoinRequested {
     }
 }
 
+#[derive(Clone, Debug)]
+#[cfg_attr(feature = "serde", derive(Serialize, Deserialize))]
+pub struct GameRichPresenceJoinRequested {
+    /// If you are joining a friend/being invited from a friend, this `SteamId` will be of said friend.
+    /// If it's not coming from a friend, this `SteamId` will be invalid, which you can check with the `.is_invalid()`
+    /// method.
+    pub friend_steam_id: SteamId,
+    /// the connect string, holding custom data to join a game or friend
+    pub connect: String,
+}
+
+unsafe impl Callback for GameRichPresenceJoinRequested {
+    const ID: i32 = CALLBACK_BASE_ID + 37;
```

Comment:
Try to use the actual symbol in steamworks-sys to tie this in the off chance this changes for whatever reason.